content.md

```diff
@@ -105,3 +108,5 @@ These are some of the bots that can be used in the in-game chat. Find the accoun
 - [osu! discord communities](https://osu.ppy.sh/wiki/en/Community/Discord_servers#official-osu!-server) - List of various discord servers made by the osu! community
 - [osu!mania Pattern Repository](https://imgur.com/a/replacement-glossary-osu-mania-pattern-repository-qEwP37W) - A glossary of common osu!mania patterns and terms
 - [Alexaaro's Imagemap Generator](https://alexaario.github.io/osu-BBCodeImagemapGenerator/) - A tool to create BBCode for imagemaps
+- [osu! cleaner](https://github.com/TCNOco/TcNo-osu-Cleaner) - Tool to save space by deleting unused elements from beatmaps
```

Comment:
Maybe I'm being picky but "unused" isn't really the best wording here? It lets the user decide what they don't want so wording it around that concept would be better than just "unused elements".

Feel free to disregard if you don't agree.

Reply:
Maybe change it to “unused/unwanted”, or just “unwanted” ?
Is there any checks in place to check whether the files are being used by the map at all?

Reply:
How about "unnecessary"?